packages/alfa-cypress/src/cypress.ts

```diff
@@ -52,21 +63,31 @@ export namespace Cypress {
     rules: Iterable<act.Rule<Page, T, Q>>,
     handlers: Iterable<Handler<Page, T, Q>> = [],
     options: Asserter.Options = {}
-  ): void {
+  ): globalThis.Chai.ChaiPlugin {
     const asserter = Asserter.of(rules, handlers, options);
 
-    const command = (value: Type) =>
-      cy.wrap(value, { log: false }).should(() => {
-        const input = toPage(value);
+    return (chai) => {
+      chai.Assertion.addMethod("accessible", async function () {
+        const input = toPage(this._obj);
 
-        const error = asserter.expect(input).to.be.accessible().get();
+        const error = await asserter.expect(input).to.be.accessible();
```

Comment:
Shouldn't that rather be called `result`?
All the `error.isOk` tests are confusing to read 😕

Reply:
Good point!